### PR TITLE
[bitami/rabbitmq] Added ability to enable/disable allocateLoadBalancerNodePorts

### LINKED ISSUE #20495: Add option to disable Load Balancer Node Port Allocation
### Name and Version

bitnami/rabbitmq 12.3.0

### What is the problem this feature will solve?

When using the LoadBalancer service type with the RabbitMQ helm chart, NodePorts are assigned for each service port. 

Being able to set `spec.allocateLoadBalancerNodePorts` to `false `would address this.

### What is the feature you are proposing to solve the problem?

`service.allocateLoadBalancerNodePorts: false`

### What alternatives have you considered?

Creating a loadbalancer manually

### PR DESCRIPTION
**Pull request by belps96**
### Description of the change

Now allows the allocateLoadBalancerNodePorts attribute of the RabbitMQ service to be disabled (normally always enabled)

### Benefits

This will allow only the desired/specified ports to be allocated a NodePort 

### Possible drawbacks

None

### Applicable issues

- fixes #20495

### Additional information
 None

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/rabbitmq/Chart.yaml [hunk 1]
```diff
@@ -30,4 +30,4 @@ maintainers:
 name: rabbitmq
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq
-version: 12.3.0
+version: 12.3.1
```
Comment (joancafom) [inline bitnami/rabbitmq/Chart.yaml:33]: As this is a new feature, it should be a minor bump :) 
```suggestion
version: 12.4.0
```

File: bitnami/rabbitmq/templates/svc.yaml [hunk 2]
```diff
@@ -26,6 +26,9 @@ spec:
   {{- if .Values.service.loadBalancerSourceRanges }}
   loadBalancerSourceRanges: {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
   {{- end }}
+  {{- if .Values.service.allocateLoadBalancerNodePorts }}
+  allocateLoadBalancerNodePorts: {{ .Values.service.allocateLoadBalancerNodePorts }}
+  {{- end }}
```
Comment (joancafom) [inline bitnami/rabbitmq/templates/svc.yaml:31]: This only happens when service is `LoadBalancer`, you will need to add a check here as done in L22.

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your PR! I left you some comments as a review

File: bitnami/rabbitmq/templates/svc.yaml (hunk 2 above)
Comment (belps96) [reply bitnami/rabbitmq/templates/svc.yaml:31]: Is this not nested within L22's if condition? Notice the {{- end }} at line 32

Comment (joancafom) [reply bitnami/rabbitmq/templates/svc.yaml:31]: You are right! I missed the `{{-end} }` at line 32 :) Just a follow-up suggestion, I don't think we need the conditional here, .if `Values.service.allocateLoadBalancerNodePorts=false` then it's okay to have `allocateLoadBalancerNodePorts=false`  (once we have ensured the service type is LoadBalancer)

```suggestion
  allocateLoadBalancerNodePorts: {{ .Values.service.allocateLoadBalancerNodePorts }}
```

**Comment by joancafom** [comment]
Just a small follow-up comment to go 😁

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your changes 🚀

**Pull request merged**